src/translations.php

```diff
@@ -219,6 +219,14 @@
         "Longest Week Streak" => "Самая длинная серия недель",
         "Present" => "Сейчас",
     ],
+    "sn" => [
```

Comment:
Looks like `sa` should be the code

https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=sa

```suggestion
    "sa" => [
```